projects/1.19/assets/caverns-and-chasms/caverns_and_chasms/lang/zh_cn.json

```
@@ -0,0 +1,411 @@
+{
+  "advancements.caverns_and_chasms.adventure.dont_move.description": "它看见你了",
+  "advancements.caverns_and_chasms.adventure.dont_move.title": "别，动",
+  "advancements.caverns_and_chasms.adventure.smelt_copper.description": "冶炼出一块铜锭",
+  "advancements.caverns_and_chasms.adventure.smelt_copper.title": "抓捕铜伙",
+  "advancements.caverns_and_chasms.adventure.summon_copper_golem.description": "将避雷针放置在雕刻过的南瓜顶部以建造一只铜傀儡",
+  "advancements.caverns_and_chasms.adventure.summon_copper_golem.title": "机械姬",
+  "advancements.caverns_and_chasms.adventure.tune_a_fish.description": "用音叉攻击一条鱼",
+  "advancements.caverns_and_chasms.adventure.tune_a_fish.title": "但你不能抓住一条鱼！",
+  "advancements.caverns_and_chasms.adventure.use_tuning_fork.description": "使用音叉从音符盒中采集音高",
+  "advancements.caverns_and_chasms.adventure.use_tuning_fork.title": "你可以给钢琴调音...",
+  "advancements.caverns_and_chasms.husbandry.netherite_hoe.description": "用下界合金锭或死灵合金锭升级一把锄，然后重新考虑你的人生抉择",
+  "advancements.caverns_and_chasms.husbandry.obtain_ancient_hoes.description": "用下界合金锭和死灵合金锭各升级一把锄，然后意识到生命是如此的微不足道",
+  "advancements.caverns_and_chasms.husbandry.obtain_ancient_hoes.title": "危险决心",
+  "advancements.caverns_and_chasms.husbandry.wax_off.description": "将蜜脾涂到铜块或铜傀儡上！",
+  "advancements.caverns_and_chasms.husbandry.wax_on.description": "给铜块或铜傀儡脱蜡！",
+  "advancements.caverns_and_chasms.nether.necromium_armor.description": "获得一整套的死灵合金盔甲",
+  "advancements.caverns_and_chasms.nether.necromium_armor.title": "诅咒缠身",
+  "advancements.caverns_and_chasms.nether.ride_boat_with_deeper.description": "在下界与深行苦力怕同船共渡",
+  "advancements.caverns_and_chasms.nether.ride_boat_with_deeper.title": "勇往深行",
+  "attribute.caverns_and_chasms.name.generic.experience_boost": "经验增益",
+  "attribute.caverns_and_chasms.name.generic.lifesteal": "生命窃取",
+  "attribute.caverns_and_chasms.name.generic.magic_damage": "魔法伤害",
+  "attribute.caverns_and_chasms.name.generic.magic_protection": "魔法保护",
+  "attribute.caverns_and_chasms.name.generic.slowness_infliction": "迟缓施加",
+  "attribute.caverns_and_chasms.name.generic.weakness_aura": "虚弱光环",
+  "block.caverns_and_chasms.amethyst": "紫水晶块",
+  "block.caverns_and_chasms.azalea_beehive": "杜鹃木蜂箱",
+  "block.caverns_and_chasms.azalea_boards": "杜鹃木砧板",

```

```suggestion
  "block.caverns_and_chasms.azalea_boards": "杜鹃木直纹板",
```
库内Woodworks使用翻译，请全文通查

---

> 库内Woodworks使用翻译，请全文通查

C&C以及悠然一派都改好了